metadata.json

```diff
@@ -3,7 +3,7 @@
   "version": "3.0.0",
   "author": "Erik Dalen <erik.gustav.dalen@gmail.com>",
   "summary": "Query functions for DNS",
-  "license": "Expat License",
+  "license": "MIT",
```

Comment:
Is this really the same license?

Reply:
https://github.com/voxpupuli/puppet-dnsquery/blob/master/LICENSE is the [MIT licence text](https://opensource.org/licenses/MIT)